utils/scripts/gen_onnx_rnn_model.py

```diff
@@ -173,6 +173,7 @@ def compute_rnn(forward):
         Wi, Ri, bWi, bRi = get_weights(dir_idx)
 
         def f(x): return np.tanh(x)
+
```

Comment:
Unrelated change?

Reply:
Yes, not directly related, but these cleanups and 32b dim_t fixes which were needed to make ninja check pass and 'format.sh fix' pass without changes made to the tree. There is a separate commit for these fixes, the new feature is in a commit after it.